doc/man7/EVP_KDF-KB.pod

```diff
@@ -0,0 +1,111 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF-KB - The Key-Based EVP_KDF implementation
+
+=head1 DESCRIPTION
+
+The EVP_KDF-KB algorithm implements the Key-Based key derivation function
+(KBKDF).  KBKDF derives a key from repeated application of a keyed MAC to an
+input secret (and other optional values).
+
+=head2 Identity
+
+"KBKDF" is the name for this implementation; it can be used with the
+EVP_KDF_fetch() function.
+
+=head2 Supported parameters
+
+The supported parameters are:
+
+=over 4
+
+=item B<OSSL_KDF_PARAM_PROPERTIES> ("properties") <UTF8 string>
+
+=item B<OSSL_KDF_PARAM_DIGEST> ("digest") <UTF8 string>
```

Comment:
Is "mac" appropriate here too?
I think SSKDF uses one.

Reply:
SSKDF does use the MAC parameter, but for toggling between HMAC and KMAC.  This makes sense for KBKDF as well since SP800-108 also defines CMAC, but I didn't implement that.  Were we to do so, we would use MAC for toggling between the two.

Unless I'm missing something about the interface, of course.

Reply:
If you're suggesting that we add the parameter now, I also wouldn't be opposed to that.

Reply:
I was suggesting adding the mac parameter.

Reply:
If adding the CMAC flavour is a lot of work, it could be done a separate PR and the MAC option would make sense there.

Reply:
I've added the MAC parameter.  I'm not sure how much work CMAC would be, but I didn't do it because I don't have a use for it.  (krb5 with camellia does use CMAC, but it's with feedback mode, not counter mode - which presumably would be another toggle, along with double-pipeline if that were implemented.)